public/lessons/graph-theory-intro/graphs-in-code.mdx

```diff
@@ -0,0 +1,205 @@
+# Representing Graphs in Code
+
+Once we have a conceptual understanding of a problem as a graph, we need to represent the graph in
+code. There are three common ways to represent graphs in code: **adjacency list**, **adjacency
+matrix** and **edge list**. Each representation has its ups and downs, and the choice of
+representation will depend on the problem at hand. However, in general, adjacency lists are
+preferred for sparse graphs (few edges), while adjacency matrices are preferred for dense graphs.
+Edge lists may be more suitable for certain algorithms.
+
+Here is an example graph to help illustrate the three representations:
+
+<Graph
+  id="graph"
+  data={{
+    nodes: [
+      { id: 0 }, 
+      { id: 1 }, 
+      { id: 2 }, 
+      { id: 3 },
+      { id: 4 },
+    ],
+    edges: [
+      { source: 0, target: 1},
+      { source: 0, target: 2},
+      { source: 0, target: 3},
+      { source: 0, target: 4},
+      { source: 1, target: 2},
+      { source: 2, target: 3},
+      { source: 3, target: 4},
+    ]
+  }}
+/>
+
+## Adjacency List
+
+An adjacency list is an array of lists. The size of the array is equal to the number of vertices in
+the graph. Let the array be `adj`. An entry `adj[u]` is a list that contains all the vertices that
+are adjacent to vertex `u`. For weighted graphs, the list can contain pairs of vertices and weights.
+
+Here is the adjacency list representation for the graph shown above:
+
+
+<div style={{ display: 'flex', justifyContent: 'center'}}>
```

Comment:
fyi: you can also wrap around `<center></center>` tag

Reply:
this doesn't work for bullet points. i think this is the same as textAlign: 'center'.

![image](https://github.com/ofast-team/frontend/assets/46213673/95171b49-1425-4c94-be24-dfa81e4b3537)

Reply:
yeah its align text, not bullet points